ci/pipeline.yml

```diff
@@ -16,6 +16,14 @@ groups:
   - prometheus-down-check-production
 
 jobs:
+- name: set-self
```

Comment:
Should this block other jobs?

Reply:
Hm, yes — I'd assumed it would be default, a poor assumption from other systems. Good catch; I'll update it.